assignments/assignment02.rb

```diff
@@ -25,11 +40,26 @@ def to_sentence(ary)
 
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
-  # your implementation here
+  ary.inject(:+) / ary.length.to_f
 end
 
 def median(ary)
-  # your implementation here
+  # If there are two middle numbers, they must be averaged
+  if ary.length % 2 == 0
+    sorted_array = ary.sort
+    last_index = sorted_array.length - 1
+    middle_index_1 = sorted_array.length / 2
+    middle_index_2 = ( last_index / 2 ).floor
+    average_of_middles = ( sorted_array[middle_index_1].to_f + sorted_array[middle_index_2].to_f ) / 2
+    # No need to return a float if it's really just an integer...
+    if ( average_of_middles % 1 ) == 0
```

Comment:
This would be a good place to use the ternary ?: operator like so:
( average_of_middles % 1 ) == 0  ? average_of_middles.to_i : average_of_middles

Also, no need for the explicit return